auth-web/src/routes/router.ts

```diff
@@ -509,6 +510,13 @@ export function getRoutes (): RouteConfig[] {
       component: TermsOfServiceView,
       meta: { requiresAuth: true }
     },
+    {
+      path: '/PAD-terms-and-conditions',
```

Comment:
the approach has to be revised so that all the kind of tos documents can be exposed as urls... may be accept the type in the url and rename it as termsofconditions view or soemthing...
since the url wont be used now , good to merge

Reply:
Might need to adjust styling and format to the terms if we do this way.